client/src/modules/dashboard/Chapters/pages/ChaptersPage.tsx

```diff
@@ -50,28 +90,51 @@ export const ChaptersPage: NextPageWithLayout = () => {
   const isLoading = loading || loadingUser || !data;
   if (isLoading || error) return <DashboardLoading error={error} />;
 
+  const filteredChapters = data.dashboardChapters.filter(({ name }) =>
+    name.toLowerCase().includes(search.toLowerCase().trim()),
+  );
+
   return (
     <VStack>
-      <Flex w="full" justify="space-between">
+      <Grid w="full" gap="1em" gridTemplateColumns="1fr 2fr 8em">
         <Heading data-cy="chapter-dash-heading" id="page-heading">
           Chapters
         </Heading>
+        <Text srOnly>
+          Type the name of chapter, that you are looking for, in the FilterBar
+          to find it.
+        </Text>
+        <Input
+          width="full"
+          type="text"
+          backgroundColor="gray.50"
+          placeholder="Filter Chapter..."
+          value={search}
+          aria-Label="FilterBar"
```

Comment:
```suggestion
          aria-label="FilterBar"
```
Is this necessary? My screen already reads the placeholder when I focus it, it seems unhelpful to repeat that info. That said it reads "Filter Chapter dot dot dot" which isn't great. If it could just say Filter Chapter, that would be perfect.

Reply:
```suggestion
```